artifactory/commands/transferconfig/transferconfig.go

```diff
@@ -122,11 +136,17 @@ func (tcc *TransferConfigCommand) Run() (err error) {
 		return
 	}
 
+	continueTransfer, err = tcc.printWarnings(sourceServicesManager)
```

Comment:
This line is duplicated in the code.

Reply:
This code was written by Yahav, I can't delete this without making sure of what he intended